### PR TITLE
Replace unmaintained cchardet with charset-normalizer

### PR DESCRIPTION
**Pull request by ttys0dev**
The cchardet package doesn't properly support python 3.11.

### CODE REVIEW

File: tests/examples/opinions/united_states/nh_example_4.compare.json [hunk 1]
```diff
@@ -201,7 +201,7 @@
   },
   {
     "case_dates": "2015-09-22",
-    "case_names": "Town of Bartlett v. Edward C. Furlong, III d/b/a Lil\u00e2\u20ac\u2122 Man Snowmobile Rentals",
+    "case_names": "Town of Bartlett v. Edward C. Furlong, III d/b/a Lil' Man Snowmobile Rentals",
```
Comment (ttys0dev) [inline tests/examples/opinions/united_states/nh_example_4.compare.json:204]: Switching to charset-normalizer also seemed to fix the output for this test.

**Comment by mlissner** [comment]
Thanks. I hadn't heard of this package. Good to shed unmaintained stuff.

**Comment by ttys0dev** [comment]
Think you could spin a new release so that I can update this in courtlistener? I think it's the last remaining blocker for python 3.11 support.

**Comment by albertisfu** [comment]
Juriscraper release is done.

**Pull request merged**